views/points-leaderboard.pug

```diff
@@ -0,0 +1,36 @@
+.row#points-board-card
```

Comment:
these leader board files seem to be very much like each other, the functionality should probably be extracted into a helper component and used through `include` functionality

Reply:
I'll clean them up